CONTRIBUTING.md

```diff
@@ -212,3 +217,55 @@ poetry export --without-hashes --format=requirements.txt > requirements.txt
 and committing the new version of the particular file into your feature
 branch. Please note that this file is only used for GitHub workflows, otherwise
 we utilize poetry for dependency  handling.
+
+
+## Recent changes
+Some changes have been made to the style guide:
```

Comment:
I think all the conventions applied in this pull request could be mentioned, i.e.

```suggestion
Some changes have been made to the style guide:
- Use `numbers.Number` as the type when both floats and integers are accepted by functions:
```python
from numbers import Number

def func(int_or_float: Number):
    ...
```
```